### DISCUSSION TITLE
Firefox updates?

### DISCUSSION THREAD

**Discussion by vulpesfelix** [sawyerpollard/Minecraft-Weather-New-Tab#22] [General]
Is the extension receiving updates only on Chrome or nowhere at all?

**Comment by GreatSymphonia** [comment]
Since the initial release, there were no updates I think.
When I'll have done the localization, @sawyerpollard will most likely update both the firefox and the chrome-based versions.

**Reply by vulpesfelix** [reply]
I see. Is there any way I could help with localization? I speak Italian but I don't have any experience in coding

**Reply by GreatSymphonia** [reply]
In about 2-3 weeks, I would likely be done with the implementation of the localization in the guidelines of google extensions.
In between, you could maybe translate the strings in the document localization.json in #12 but the format in which those strings will be implemented will differ.

**Comment by sawyerpollard** [comment]
> Is the extension receiving updates only on Chrome or nowhere at all?

@Frantee Hi there! So, I haven't actually submitted any new updates to the Chrome Store yet. Though, I think there are now enough good updates that it's worth submitting them.

In regards to translations, I've been able to localize the extension with Chrome's i18n API. If you want to make Italian translations, all you have to do is translate this file: https://github.com/sawyerpollard/Minecraft-Weather-New-Tab/blob/master/src/_locales/en/messages.json

Only translate the text in the "message" fields. And also don't translate text between $, like $TEMPERATURE$.

That's it. And it will show up in Italian based on your language setting for Chrome.

Let me know if you need help.

**Reply by vulpesfelix** [reply]
Thanks for the answer!
Will it work only for Chrome or Firefox as well?

**Reply by sawyerpollard** [reply]
Yep, it should.

https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/i18n

**Reply by vulpesfelix** [reply]
Just made a pull request :)

**Comment by sawyerpollard** [comment]
I'm excited to update the Firefox version, but I'm a little worried about the DMCA issue I had on Chrome. I'm taking steps though to make sure I can update it.

**Reply by vulpesfelix** [reply]
that's okay, take your time. In the meantime, how can I load the extension from source on Firefox? I think I can only load it as a temporary extension for testing but can't figure out how.
